config/val.config.src

```diff
@@ -82,6 +84,11 @@
                       header_table_size => 4096,
                       enable_push => 1,
                       max_concurrent_streams => unlimited,
+                      max_connections => 2500,                   %% the max connections permitted, after which connections will be refused
```

Comment:
This seems low? I'm not sure what kind of numbers we normally see

Reply:
that does seem low since there's durable connections as well as the spew of connections as part of challenge handling

Reply:
i have bumped to 4k connections here ca29e9f79c837386f36ed16d1f32c9fcc37cc283. 

Looking back on val ops channel, the spikes do seems to be around the 2.5k mark.

If you consider 1m hotspots spread across 1000 vals ( even tho we have much more than that ) it works out average of 1K connections per hotspot.

But maybe best to start high and work down from there